sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -135,7 +129,7 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * @since 1.4.0
    */
   def load(path: String): DataFrame = {
-    option("path", path).load()
+    load(Seq(path): _*) // force invocation of `load(...varargs...)`
```

Comment:
i believe that works as expected (i am running into some other issues now, but they seem unrelated). 
however from a DSL perspective this is not very pretty?